homework/index.js

```diff
@@ -0,0 +1,21 @@
+const repoSelectElement = document.getElementById('repositories');
+const unContainer = document.querySelector('.right__div-username');
+const reposContainer = document.querySelector('.right__div-repos');
+const avatarContainer = document.querySelector('.right__div-avatar');
+
+
+const CONTRIBUTORS_URL = async () => {
+  const API_CALL = await fetch(
+    'https://api.github.com/repos/HackYourFuture/tdd-game/contributors',
+  );
+  const data = await API_CALL.json();
+  return { data };
+};
+const showContributorsDetails = () => {
+  CONTRIBUTORS_URL(repoSelectElement.value).then((response) => {
```

Comment:
You're passing the value of the repository select element here. Nice find! But this will contain the *name* of the repository. You must then still somehow get its contributors url.